app/controllers/topics_controller.rb

```diff
@@ -75,6 +85,21 @@ def update_topic_params
   end
 
   def updatable_topic_attributes
-    [:title, :description, :tree, content_owner_ids: []]
+    [:title, :description, content_owner_ids: []]
+  end
+
+  def store_topic_sections_in_topic
+    @topic.topic_sections.destroy_all
```

Comment:
Will the related `TopicSectionGuide`s get blown away with this? I can't see the dependent: :destroy but I might be missing something.